draft-ietf-core-yang-cbor-latest.md

```diff
@@ -1681,36 +1716,40 @@ Author/Change controller:
 ## CoAP Content-Formats Registry
 
 This document adds the following Content-Format to the "CoAP Content-Formats",
-within the "Constrained RESTful Environments (CoRE) Parameters" registry.
-
-| Media Type                             | Content Coding | ID   | Reference |
-| application/yang-data+cbor; id=name    |                | TBD1 | RFC XXXX  |
+within the "Constrained RESTful Environments (CoRE) Parameters"
+registry, where TBD3 comes from the "Expert Review" 0-255 range and
+TBD1 and TBD2 come from the "IETF Review" 256-9999 range.
+
+| Media Type                          | Content Coding | ID   | Reference |
+| application/yang-data+cbor          |                | TBD1 | RFC XXXX  |
```

Comment:
I think we usually put a `-` as the Content Coding in RFCs to make it obvious that it has been filled out. (The IANA registry then leaves it empty, though.)

Reply:
Fixed.